install/Knative-with-Minishift.md

```diff
@@ -164,13 +164,14 @@ curl -s https://raw.githubusercontent.com/knative/docs/master/install/scripts/is
 1. Run the following to install Istio:
 
    ```shell
+   kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.2/istio-crds.yaml && \
    oc apply -f https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml
    ```
-
-   > **NOTE:** If you get a lot of errors after running the above command like:
-   > **unable to recognize "STDIN": no matches for kind "Gateway" in version
-   > "networking.istio.io/v1alpha3"**, just run the command above again, it's
-   > idempotent and hence objects will be created only once.
+   Note: the resources (CRDs) defined in the `istio-crds.yaml`file are
+   also included in the `istio.yaml` file, but they are pulled out so that
+   the CRD definitions are created first. If you see an error when creating
+   resources about an unknown type, run the second `kubectl apply` command
+   again.
```

Comment:
While this file already had a variant of this text, I opted to replace it with the new stuff for consistency.